diff --git a/node_modules/@polkadot/react-identicon/Identicon.js b/node_modules/@polkadot/react-identicon/Identicon.js
index d9f86c2..8294a13 100644
--- a/node_modules/@polkadot/react-identicon/Identicon.js
+++ b/node_modules/@polkadot/react-identicon/Identicon.js
@@ -7,7 +7,7 @@ import { ICON_DEFAULT_HOST, settings } from '@polkadot/ui-settings';
 import { isHex, isU8a, u8aToHex } from '@polkadot/util';
 import { decodeAddress, encodeAddress, ethereumEncode } from '@polkadot/util-crypto';
 import { Beachball, Empty, Ethereum, Jdenticon, Polkadot } from "./icons/index.js";
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 const Fallback = Beachball;
 const DEFAULT_SIZE = 64;
 const Components = {
diff --git a/node_modules/@polkadot/react-identicon/cjs/icons/Beachball.js b/node_modules/@polkadot/react-identicon/cjs/icons/Beachball.js
index 630ab33..a7e9f5c 100644
--- a/node_modules/@polkadot/react-identicon/cjs/icons/Beachball.js
+++ b/node_modules/@polkadot/react-identicon/cjs/icons/Beachball.js
@@ -9,7 +9,7 @@ var _react = _interopRequireWildcard(require("react"));
 
 var _uiShared = require("@polkadot/ui-shared");
 
-var _jsxRuntime = require("react/jsx-runtime");
+var _jsxRuntime = require("react/jsx-runtime.js");
 
 function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function (nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }
 
diff --git a/node_modules/@polkadot/react-identicon/cjs/icons/Empty.js b/node_modules/@polkadot/react-identicon/cjs/icons/Empty.js
index 0458874..4e7aec4 100644
--- a/node_modules/@polkadot/react-identicon/cjs/icons/Empty.js
+++ b/node_modules/@polkadot/react-identicon/cjs/icons/Empty.js
@@ -9,7 +9,7 @@ exports.Empty = void 0;
 
 var _react = _interopRequireDefault(require("react"));
 
-var _jsxRuntime = require("react/jsx-runtime");
+var _jsxRuntime = require("react/jsx-runtime.js.js");
 
 // Copyright 2017-2022 @polkadot/react-identicon authors & contributors
 // SPDX-License-Identifier: Apache-2.0
diff --git a/node_modules/@polkadot/react-identicon/cjs/icons/Ethereum.js b/node_modules/@polkadot/react-identicon/cjs/icons/Ethereum.js
index ddda67a..d8a9785 100644
--- a/node_modules/@polkadot/react-identicon/cjs/icons/Ethereum.js
+++ b/node_modules/@polkadot/react-identicon/cjs/icons/Ethereum.js
@@ -13,7 +13,7 @@ var _react = _interopRequireWildcard(require("react"));
 
 var _styledComponents = _interopRequireDefault(require("styled-components"));
 
-var _jsxRuntime = require("react/jsx-runtime");
+var _jsxRuntime = require("react/jsx-runtime.js");
 
 function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function (nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }
 
diff --git a/node_modules/@polkadot/react-identicon/icons/Beachball.js b/node_modules/@polkadot/react-identicon/icons/Beachball.js
index 99981e8..852474c 100644
--- a/node_modules/@polkadot/react-identicon/icons/Beachball.js
+++ b/node_modules/@polkadot/react-identicon/icons/Beachball.js
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 import React, { useCallback } from 'react';
 import { beachballIcon } from '@polkadot/ui-shared';
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 
 function Identicon({
   address,
diff --git a/node_modules/@polkadot/react-identicon/icons/Empty.js b/node_modules/@polkadot/react-identicon/icons/Empty.js
index 8deaac8..8443d91 100644
--- a/node_modules/@polkadot/react-identicon/icons/Empty.js
+++ b/node_modules/@polkadot/react-identicon/icons/Empty.js
@@ -1,7 +1,7 @@
 // Copyright 2017-2022 @polkadot/react-identicon authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 
 function Identicon({
   className = '',
diff --git a/node_modules/@polkadot/react-identicon/icons/Ethereum.js b/node_modules/@polkadot/react-identicon/icons/Ethereum.js
index dd39150..4638f93 100644
--- a/node_modules/@polkadot/react-identicon/icons/Ethereum.js
+++ b/node_modules/@polkadot/react-identicon/icons/Ethereum.js
@@ -3,7 +3,7 @@
 import makeBlockie from 'ethereum-blockies-base64';
 import React, { useMemo } from 'react';
 import styled from 'styled-components';
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 
 function Identicon({
   address,
diff --git a/node_modules/@polkadot/react-identicon/icons/Jdenticon.js b/node_modules/@polkadot/react-identicon/icons/Jdenticon.js
index 43cc415..ed67ff0 100644
--- a/node_modules/@polkadot/react-identicon/icons/Jdenticon.js
+++ b/node_modules/@polkadot/react-identicon/icons/Jdenticon.js
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 import * as jdenticon from 'jdenticon';
 import React, { useMemo } from 'react';
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 
 function Identicon({
   className = '',
diff --git a/node_modules/@polkadot/react-identicon/icons/Polkadot.js b/node_modules/@polkadot/react-identicon/icons/Polkadot.js
index d05c722..3779edb 100644
--- a/node_modules/@polkadot/react-identicon/icons/Polkadot.js
+++ b/node_modules/@polkadot/react-identicon/icons/Polkadot.js
@@ -15,7 +15,7 @@
 //   - Overall it is now just a static component, expecting an address as an input value
 import React, { useMemo } from 'react';
 import { polkadotIcon } from '@polkadot/ui-shared';
-import { jsx as _jsx } from "react/jsx-runtime";
+import { jsx as _jsx } from "react/jsx-runtime.js";
 
 function renderCircle({
   cx,
